public-ui/src/app/data-browser/chart/chart.component.ts

```diff
@@ -20,6 +20,7 @@ export class ChartComponent implements OnChanges {
   @Input() chartType: string;
   @Input() sources = false;
   @Input() genderId: string; // Hack until measurement design of graphs gender overlay
+  @Input() component: string;
```

Comment:
nit: `component` has too strong of a meaning in Angular, would be better to use something like `category`, or `domainType`?

Reply:
Changed it to domain type thanks.